requirements/base.txt

```diff
@@ -1,7 +1,7 @@
 bs4
```

Comment:
are these updated pins all necessary for the web3 lib upgrade?

Reply:
yes